test/plugins/pipelines.test.js

```diff
@@ -856,7 +856,7 @@ describe('pipeline plugin test', () => {
     describe('POST /pipelines', () => {
         let options;
         const unformattedCheckoutUrl = 'git@github.com:screwdriver-cd/data-MODEL.git';
-        const checkoutUrl = 'git@github.com:screwdriver-cd/data-model.git';
+        const checkoutUrl = 'git@github.com:screwdriver-cd/data-model.git#master';
```

Comment:
Yes, the `scm` will append the `master`: https://github.com/screwdriver-cd/scm-github/blob/master/index.js#L42. shouldn't need it here